data.yaml

```diff
@@ -0,0 +1,10 @@
+aws:
+  region: eu-west-1
+  keyname: aws-key-name
```

Comment:
This should probably look something like an SSH key name to give the clue that's what we're talking about.

Reply:
e.g.`my-instance-ssh.pem`

Reply:
Just killed the data.yaml instead.